.pylintrc

```diff
@@ -0,0 +1,27 @@
+[REPORTS]
+reports=no
+
+[FORMAT]
+max-line-length=120
+
+[MESSAGES CONTROL]
+disable=
+	abstract-class-not-used,
+	broad-except,
+	fixme,
+	invalid-name,
+	locally-disabled,
+	maybe-no-member,
+	protected-access,
+	star-args,
+	too-few-public-methods,
+	too-many-ancestors,
+	too-many-arguments,
+	too-many-branches,
+	too-many-locals,
+	too-many-public-methods,
+	too-many-statements,
```

Comment:
We should not disable this linter in particular. The point of that linter is not just clean code, but good code. too-many-statements means that the cyclomatic complexity of the method is very high. We should avoid that pattern & therefore it should be removed from this file so it can be enforced.